### PR TITLE
feat(chronograf): Generate secrets for Chronograf running in a docker-compose environment

### PR DESCRIPTION
**Pull request by w-le**
This PR: 
1. Extends the automated secret generation for InfluxDB and does similar for Chronograf:
  a. Save the same INFLUX API KEY value to the same .env file but with an env var key that satisfies chronograf (the existing one satisfies Source)
  b. Generate a a`TOKEN_SECRET` as required by Chrongraf v1.9 for OAuth integration
2. Add an /analytics app by default, partially automating Chronograf Oauth integration (PlaceOS as the OAuth provider)

Feature 1 in this PR is a perquisite of https://github.com/place-labs/partner-environment/pull/80

### CODE REVIEW

**Review by caspiano — DISMISSED** [review]
To prevent potential confusion, I think `TOKEN_SECRET` should be renamed to `CHRONOGRAF_TOKEN_SECRET` then mapped to the correct env var for the container in the `docker-compose.yml`

**Comment by w-le** [comment]
@caspiano yeah I had actually thought about that last night. Ideally I would see the docker-compose have
```
chronograf:
  ...
  environment:
    - TOKEN_SECRET: $CHRONOGRAF_TOKEN_SECRET
```
but for that to work, the random `CHRONOGRAF_TOKEN_SECRET` would need to be dynamically added to `.env` as opposed to being written to it's own file here and being added as an [env_file](https://github.com/place-labs/partner-environment/blob/feat/chronograf/docker-compose.yml#L385)

Do you see an easier/simpler solution to having the `placeos` script modify the .env file prior to compose up?

Another alternative is to have the `placeos` script source a `.env.chronograf` file that contains `CHRONOGRAF_TOKEN_SECRET` prior to docker-compose up, but the downside to that is that users running `docker-compose up` (instead of `placeos start`) won't have the `CHRONOGRAF_TOKEN_SECRET` env var loaded in order to make the first snippet above work.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: CI
+on: push
+jobs:
+  style:
```
Comment (caspiano) [inline .github/workflows/ci.yml:4]: Could you update these to use the centralised workflows? An example can be found in PlaceOS/rest-api

**Review by caspiano — CHANGES_REQUESTED** [review]
Small change required on the CI flow, but otherwise happy with this. 

My previous comments seem a bit unnecessary.

**Review by caspiano — APPROVED** [review]

**Pull request merged**